src/components/Search.js

```diff
@@ -1,12 +1,9 @@
 import React from 'react'
 
-const Search = () => {
+function Search(props) {
   return (
     <div className="ui huge fluid icon input">
-      <input
-        type="text"
-        placeholder={"Search your Recent Transactions"}
-      />
+      <input type="text" placeholder="Search your Recent Transactions" onChange={props.handleChange}/>
```

Comment:
don't forget to add a `value` prop here to the input. Also, on a real code challenge, I would probably avoid making stylistic changes like an arrow function to a named function, unless you have a specific reason to do so.